scm-ui/ui-webapp/src/admin/containers/AdminDetails.tsx

```diff
@@ -62,7 +95,32 @@ class AdminDetails extends React.Component<Props> {
       <>
         <Title title={t("admin.info.title")} />
         <NoBottomMarginSubtitle subtitle={t("admin.info.currentAppVersion")} />
-        <BottomMarginDiv>{this.props.version}</BottomMarginDiv>
+        <BottomMarginDiv>{version}</BottomMarginDiv>
+        {releaseInfo && (
+          <>
+            <BoxShadowBox className="box">
+              <article className="media">
+                <ImageWrapper className="media-left image is-96x96">
+                  <Image src="/images/blib.jpg" alt={t("admin.info.logo")} />
+                </ImageWrapper>
+                <div className="media-content">
+                  <div className="content">
+                    <h3 className="has-text-weight-medium">{t("admin.info.newRelease.title")}</h3>
+                    <p>
+                      {t("admin.info.newRelease.description", {
+                        version: releaseInfo?.version
+                      })}
+                    </p>
+                    <a className="button is-info is-pulled-right" target="_blank" href={releaseInfo?.link}>
```

Comment:
I think I would make this a warning
```suggestion
                    <a className="button is-warning is-pulled-right" target="_blank" href={releaseInfo?.link}>
```